### PR TITLE
Rewrite manual page using mdoc(7) markup language

### PR DESCRIPTION
**Pull request by czarkoff**
Mdoc provides semantic markup, which allows better results when rendering manual
to other formats.

### CODE REVIEW

**Comment by eXeC64** [comment]
Nice. This looks a lot cleaner, I just have a few suggestions that I'll comment onto the diff.

File: doc/imv.1 [hunk 1]
```diff
@@ -1,131 +1,125 @@
...
-imv \- image viewer
-.SH SYNOPSIS
-.nf
-\fBimv\fP [-rfaudlh] [-n <NUM|PATH>] [-b COLOR] [-e FONT:SIZE] [-t SECONDS] [images...]
-.fi
-.sp
-.SH DESCRIPTION
-.sp
-\fBimv\fP is a X11/Wayland image viewer aimed at users of tiling window managers.
+.Dd $Mdocdate: December 30 2015$
```
Comment (eXeC64) [inline doc/imv.1]: Viewing the new manpage using man 2.7.5 displays the `$` after `2015`. I'm also seeing "BSD General Commands Manual" written to the header, and "BSD" added to the footer. Is this an assumption the man page reader is making owing to the syntax used?

File: doc/imv.1 [hunk 2]
```diff
@@ -1,131 +1,125 @@
...
-.TP
-.B -r
-Recursively search input paths for files.
-.TP
-.B -f
-Start in full screen mode
-.TP
-.B -a
+.Nm
+reloads current image if it detects changes to the file.
```
Comment (eXeC64) [inline doc/imv.1]: Missing a `the` in here: `reloads the current image`

File: doc/imv.1 [hunk 3]
```diff
@@ -1,131 +1,125 @@
...
+or a hex color value.
+Defaults to
+.Ar #000000 .
+.It Fl e Ar font:size
+Set the font used by the overlay.
+.Ar font
+can be any valid system font, such as FreeSans, FreeMono, etc.
+Defaults to
+.Ar Monospace:24 ,
+which is default monospace font, as configured in
```
Comment (eXeC64) [inline doc/imv.1]: Another `the`: `which is the default`

File: doc/imv.1 [hunk 4]
```diff
@@ -1,131 +1,125 @@
...
+.It Fl t Ar seconds 
+Set the slideshow delay in seconds.
+Setting this to zero disables slideshow mode, which is the default.
+.It Fl u
+Use nearest neighbour resampling. Recommended for pixel art.
+.El
+.Sh CONTROLS
+.Bl -tag -width Ds
+.It Aq Cm left mouse button
+Pan in drag direction.
```
Comment (eXeC64) [inline doc/imv.1]: Very nitpicky here, but if I click and drag left, I pan right. I'm not sure if "in drag direction" is an apt description.

Perhaps "Pan relative to mouse motion" or similar would be more correct?

**Comment by eXeC64** [comment]
Other than the points raised above this looks good. The man page is a lot more readable. Thanks.

File: doc/imv.1 (hunk 1 above)
Comment (czarkoff) [inline doc/imv.1]: The '$' sign is my mistake - I'm too much used to CVS expansion.

"BSD General Commands Manual" in the header is expanded from default name of section 1 of your operating system. "BSD" in the footer is expanded from your operating system name. Apparently your groff is misconfigured. What distro do you use?

Comment (eXeC64) [inline doc/imv.1]: I'm running Arch Linux. Out of curiosity what are you running? FreeBSD 11?

Comment (czarkoff) [inline doc/imv.1]: OpenBSD -current.

**Pull request merged**